cmd/boot-script-service/oauth.go

```diff
@@ -51,6 +55,29 @@ func (nc nowClock) Now() time.Time {
 	return time.Now()
 }
 
+func fetchPublicKey(url string) error {
```

Comment:
I think it would be useful to have a short descriptive comment for this function too.

Reply:
Resolved in [669a4b7](https://github.com/OpenCHAMI/bss/pull/26/commits/669a4b77b68f7500793b264b306a5d27a34c9413)